Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -657,6 +669,11 @@ WarpX::PushPSATD ()
 
     const int rho_old = spectral_solver_fp[0]->m_spectral_index.rho_old;
     const int rho_new = spectral_solver_fp[0]->m_spectral_index.rho_new;
+    const ablastr::fields::MultiLevelScalarField rho_fp = m_fields.get_mr_levels("rho_fp", finest_level);
+    const ablastr::fields::MultiLevelScalarField rho_cp = m_fields.get_mr_levels("rho_fp", finest_level);
```

Comment:
Just found another typo bug:
```suggestion
    const ablastr::fields::MultiLevelScalarField rho_fp = m_fields.get_mr_levels("rho_fp", finest_level);
    const ablastr::fields::MultiLevelScalarField rho_cp = m_fields.get_mr_levels("rho_cp", finest_level);
```
Interesting that CI didn't catch it...

Reply:
that's actually quite alarming! This means that we should probably add/modify a test in such a way that it would have caught this...

Reply:
It means our test coverage is not great and does not cover this logic.